src/parse-transaction-receipts.ts

```diff
@@ -94,17 +96,97 @@ export default function parseTransactionReceipts(
           assert(isHealthCheckEvent(event));
 
           result.push({
-            block: Number(receipt.blockNumber),
+            blockNumber,
             event,
-            txHash: log.transactionHash,
+            txHash,
+          });
+          break;
+        }
+
+        case EventType.ProposalQueued: {
+          const event = decodeEventLog({
+            abi: GovernorABI,
+            data: log.data as `0x${string}`,
+            topics: log.topics as [
+              signature: `0x${string}`,
+              ...args: `0x${string}`[],
+            ],
+          });
```

Comment:
Can remove some of these duplicates if you want, as it's the same for all events except for `MedianUpdated`